### DISCUSSION TITLE
中國遊戲平台利用AI臉部辨識監控兒童遊玩時間

### DISCUSSION THREAD

**Discussion by ChinChun0220** [1101-GS4719-C/Python-Community-GS4719-C#253] [科技新知]
這個蠻久之前的新聞了，不確定前面有沒有同學po過。
去年中國提出一個政策要限制未成年兒童遊玩手機遊戲的時間，其中提出非常多種措施（ex需要身份證登入等），但漏洞百出，因此開發一個新的方法，在遊玩時開啟人臉辨識系統確認是否是成人遊玩，若不是則鎖遊戲。

報導原文：https://9gag.com/gag/aEpQNxG?fbclid=IwAR2x9aEDfmxUQEnV5a0Sc9JTeJehaBr60Oyyx3KeAWjN6iSCk913oNtVH_M

出發點是好的，但時不時的監控應該會導致很嚴重的隱私權問題？？ 看網站寫是已經在測試，蠻好奇中國民眾是否反對這項措施… 每次遇到這種議題都會想看看中國民眾的討論，但都找不到地方。

**Comment by pandaiscollin** [comment]
在中國應該沒有隱私的問題吧,民眾就算反對也沒啥用,沒多久就又被壓下去了。

**Comment by zixunn** [comment]
感覺就很像中國已經存在的天眼系統。

**Reply by ChinChun0220** [reply]
天眼照的是公共區域，道路之類的，手機鏡頭屬於很隱私的，感覺不太一樣。
當初iphone 的Siri 收音的爭議都鬧超大的了…

**Comment by 109302003** [comment]
感覺隱私權、版權、著作權在中國都是沒有的事。

**Comment by yt7l0** [comment]
如果可以在玩遊戲把鏡頭打開，那怎麼知道平常使用手機的時候鏡頭是不是關著的呢，隨時處在被監控的恐懼裡真的很恐怖……
然後找不到地方看他們討論應該是正常的，感覺他們也找不到地方討論QQ

**Comment by YuHengChen54** [comment]
中國人真的沒有隱私欸，甚麼都監控

**Comment by 109302013evantsao** [comment]
總結就是輕犯隱私了，還是沒有起到限制兒童遊玩遊戲的目的，我不認為這些兒童會乖乖放下手機，去念書，就像在侏儸紀公園裡我最愛的那句話一樣，生命總會自己找到出路，它們也一定可以找到漏洞鑽

**Comment by charlottechuang** [comment]
這樣真的很沒隱私... 認為控管孩童使用手機等電子產品的方式有很多，例如上課的時候集中保管（？） 當然更好的是想辦法教孩子自律，現在家長太寵小孩了，才會衍生出很多小孩因為不能玩手機就生氣怎樣的問題。

**Comment by www0000596** [comment]
呼應前陣子的一篇文章，中國利用監控個人信用機制的措施...看來監控個人日常行為這件事似乎不是不可能了...
